modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/GroupPageStoresMap.java

```diff
@@ -90,7 +90,7 @@ public boolean contains(GroupPartitionId groupPartitionId) {
      * Returns a view of all page stores of all groups.
      */
     public Collection<GroupPartitionPageStore<T>> getAll() {
-        return CollectionUtils.viewReadOnly(groupPartitionIdPageStore.entrySet(), GroupPartitionPageStore::new);
+        return groupPartitionIdPageStore.entrySet().stream().map(GroupPartitionPageStore::new).collect(Collectors.toList());
```

Comment:
This part was explicitly written in this way to avoid allocating new collection. Please make this method return Stream, for example. It's on the hot path.